packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -221,7 +223,7 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
       try {
         imports = parseImports(source)[0]
       } catch (e: any) {
-        this.error(e)
+        this.error(e, e.idx)
```

Comment:
This `this.error` is in the transform hook. This was removed by mistake in #13608.